pkg/cmd/env/templates/rc.tmpl

```diff
@@ -0,0 +1,57 @@
+{{define "bash" -}}
+if [ -z "$GCTL_SESSION_ID" ] && [ -z "$TERM_SESSION_ID" ]; then
+  export GCTL_SESSION_ID=$(uuidgen)
+fi
+source <(gardenctl completion bash)
```

Comment:
does the completion work? as our completion help shows something different (also for zsh and fish)

Reply:
A `compdef _gardenctl gardenctl` was missing